api/pom.xml

```diff
@@ -77,7 +77,7 @@
     <properties>
         <copyright.exclude>${project.basedir}/../etc/copyright-exclude</copyright.exclude>
         <copyright.templatefile>${project.basedir}/../etc/copyright.txt</copyright.templatefile>
-        <copyright.ignoreyear>false</copyright.ignoreyear>
+        <copyright.ignoreyear>true</copyright.ignoreyear>
```

Comment:
Copyright year update to 2022

Reply:
can you point me to the file modified in this PR which does not have updated cp year?

Reply:
This very file api/pom.xml

Reply:
ah, right, fixed